docs/available_commands.md

```diff
@@ -171,6 +171,80 @@ So only a message that the component was not found and could not be updated is l
 
     cdx-ev set bom.json --from-file mysetfile.json --ignore-missing
 
+#### set for version ranges
```

Comment:
I extended the existing chapters, but as mentioned above, feels lacking.

Reply:
I'm not so sure. Looks okay to me. You provide the core bits of info:

- You can use the `version_range` field instead of `version`.
- Ranges must conform to the *vers* specification to be found at your link.

And there are examples, too. I consider this resolved.